### PR TITLE
Adding websocket event for test status update

### PR DESCRIPTION
**Pull request by Jnegrier**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -622,6 +622,16 @@ def update_test_history():
 
     data = {"message": "Test history updated successfully"}
 
+    resolution_event = {
```
Comment (opishcheiko) [inline app.py:625]: resoluuution. should we giive it a clearer name? like test_update or something?

**Review by opishcheiko — APPROVED** [review]

File: app.py (hunk 1 above)
Comment (opishcheiko) [reply app.py:625]: even delta_test is not super clear. Or it's normal naming convention?

Comment (Jnegrier) [reply app.py:625]: Yeah, I think we could define a better name, we've full control over the convention, AFAIK there is no convention about this. We should just think about the impact in the architectural design, for instance, an event could be triggered in several different parts of the website in realtime, so maybe a generic name make sense

Comment (opishcheiko) [reply app.py:625]: okay, let's change it to test_event then?

Comment (Jnegrier) [reply app.py:625]: Sounds good to me 👍

**Comment by opishcheiko** [comment]
@Jnegrier how does that look? https://github.com/delta-reporter/delta-core/pull/42/commits/6dc966cb859a90c6d562a42aa03378fff1579e92

**Comment by Jnegrier** [comment]
> @Jnegrier how does that look? [6dc966c](https://github.com/delta-reporter/delta-core/commit/6dc966cb859a90c6d562a42aa03378fff1579e92)

Looks amazing, thanks @opishcheiko

**Pull request merged**